.github/workflows/reusable-module-prep-release.yml

```diff
@@ -13,12 +13,22 @@ on:
       module-branch:
         description: The branch of the module to use.
         type: string
-        required: true
+        default: 'main'
+        required: false
       level:
         description: The level of release to be used.
         type: string
-        default: 'patch'
         required: true
+      json-file:
+        description: The JSON file to be used.
+        type: string
+        default: 'package.json'
+        required: false
+      php-file:
+        description: The PHP file to be used.
```

Comment:
Same here, let's include more context to make this clear.

Reply:
Updated this one too